JniBridge.bee.cs

```diff
@@ -91,6 +91,7 @@ static class Platform
         "::android::view::Display",
         "::android::view::Gravity",
         "::android::view::SurfaceView",
+        "::android::view::WindowInsets_Type",
```

Comment:
Looks like some of these use underscore and some (MediaCode::BufferInfo) a double colon. Is there a difference?
I'd prefer to have WindowInsets::Type, a more direct counterpart.

Reply:
"::android::view::WindowInsets::Type" doesn't generate the API

Reply:
But what's the difference from https://developer.android.com/reference/android/media/MediaCodec.BufferInfo
Or perhaps we don't have API generated for BufferInfo?

Reply:
we do have api generated for BufferInfo, not sure - what's difference in this case. This is something we would need investigate, but obviously not now... If you want I can fill the ticket

Reply:
> If you want I can fill the ticket

Please do.

Reply:
Btw, I noticed if you remove "::android::media::MediaCodec::BufferInfo", the binding is still generated, thus maybe it's like you said - dependencies are generated, so "::android::media::MediaCodec_BufferInfo" is correct, and "::android::media::MediaCodec::BufferInfo" is incorrect.

Do you still want a ticket?

Reply:
So what is the generated class name?
If it's MediaCodec_BufferInfo, then it's all by design.

Reply:
In C++ it's android::media::MediaCodec_BufferInfo